src/github/client/github-queries.ts

```diff
@@ -163,24 +164,26 @@ export type pullRequestQueryResponse = {
 	repository: {
 		pullRequests: {
 			edges: {
+				createdAt: string;
 			cursor: string,
 			node: pullRequestNode;
 			}[]
 		};
 	};
 };
 
-export const getPullRequests = `query ($owner: String!, $repo: String!, $per_page: Int!, $commitSince: GitTimestamp, $cursor: String) {
+export const getPullRequests = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
   repository(owner: $owner, name: $repo) {
 
-		pullRequests(first: $per_page, orderBy: {field: CREATED_AT, direction: DESC}, after: $cursor, filterBy: { since: $commitSince }) {
```

Comment:
github :shakesfist: doesnt support any meaningful filtering on pulls, the only graphql way to do it is using the search() call, and i aint doing that, adds too much complexity.